python/lsst/ap/verify/metrics.py

```diff
@@ -98,15 +113,28 @@ class AutoJob:
 
     Parameters
     ----------
+    butler : `lsst.daf.persistence.Butler`
+        the repository associated with this ``Job``.
```

Comment:
I feel like you've slipped into starting your docstrings with lower case somewhere around here, and beginning each "sentence" with an uppercase letter would be more in line with our style guide.